tty/libtty/libtty_disc.c

```diff
@@ -377,29 +456,30 @@ ssize_t libttydisc_read_canonical(libtty_common_t *tty, char *data, size_t size,
 			return -EWOULDBLOCK;
 		}
 
-		if (st) { // nonblocking
-			st->timeout_ms = 0; // wait indefinitely
+		if (st) {                // nonblocking
```

Comment:
It's weird that clang-format doesn't complain here. One space before comment is enough
```suggestion
		if (st) { // nonblocking
```